src/webgpu/shader/validation/wgsl/runtime-array-not-last.fail.wgsl

```diff
@@ -1,5 +1,6 @@
 # v-0015 - This fails because of the runtime array is not last member of the struct.
 
+[[block]]
 struct Foo {
   [[offset (0)]] a : array<f32>;
```

Comment:
Isn't this also invalid because the array has no stride?

Reply:
I cannot find this in spec. Could you point my to what you're referring to?